### DISCUSSION TITLE
Installation, configuration of the latest version of The ONE Simulator.

### DISCUSSION THREAD

**Discussion by osmankhalid2005** [akeranen/the-one#121] [General]
Here I created a short tutorial of installation and configuration of the latest version of the ONE Simulator.

[How to configure the ONE Simulator
](https://www.youtube.com/watch?v=xYUjnmxpoWk)

**Comment by akeranen** [comment]
Nice! I added this to the list of [community resources](http://akeranen.github.io/the-one/#community-resources).
